cmd/tendermint/commands/reset.go

```diff
@@ -129,20 +140,13 @@ func resetState(dbDir string, logger log.Logger, keyType string) error {
 		}
 	}
 
-	if tmos.FileExists(peerstore) {
-		if err := os.RemoveAll(peerstore); err == nil {
-			logger.Info("Removed peerstore.db", "dir", peerstore)
-		} else {
-			logger.Error("error removing peerstore.db", "dir", peerstore, "err", err)
-		}
-	}
-	if err := tmos.EnsureDir(dbDir, 0700); err != nil {
-		logger.Error("unable to recreate dbDir", "err", err)
-	}
-	return nil
+	return tmos.EnsureDir(dbDir, 0700)
 }
 
-func resetFilePV(privValKeyFile, privValStateFile string, logger log.Logger, keyType string) error {
+// ResetFilePV loads the file private validator and resets the watermark to 0. If used on an existing network,
+// this can cause the node to double sign.
+// XXX: this is unsafe and should only suitable for testnets.
```

Comment:
Maybe we should have this printed to stderr every time it is run, just to drive the point home.

Reply:
Wouldn't printing to stderr be more likely to indicate that something is wrong. I think/hope we're giving sufficient warning